src/Dapr.Actors/Runtime/Actor.cs

```diff
@@ -213,9 +214,10 @@ protected virtual Task OnPostActorMethodAsync(ActorMethodContext actorMethodCont
             string reminderName,
             byte[] state,
             TimeSpan dueTime,
-            TimeSpan period)
+            TimeSpan period,
+            TimeSpan? ttl = null)
```

Comment:
This is a breaking change. See: https://github.com/ltrzesniewski/dotnet-runtime/blob/master/docs/coding-guidelines/breaking-change-rules.md#signatures

This needs to be a new overload.

Reply:
When you add the new overload, we probably can't make `ttl` optional, because that will cause an ambiguity.

Reply:
Dang, I thought the default saved me from the breaking change. Thanks for letting me know! I'll fix this and apply throughout.

Reply:
> When you add the new overload, we probably can't make ttl optional, because that will cause an ambiguity in overload resolution.

So this was a problem because null is semantically important here unfortunately. I didn't want to put in something like looking for a specific TimeSpan value made us remove the TTL because who knows what people want to put in (and if it's wrong I'd rather error). So I introduced a class that allows us to pass in a null while keeping a consistent API surface. Let me know what you think, it's generally internal because it felt weird to expose it.